packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -30,39 +69,46 @@ const propTypes = {
    * An ISO 8601 string representation of the time value in the input.
    */
   value: PropTypes.string,
+  /**
+   * Type of time input to initialize. Must be '24' or '12'
```

Comment:
nit: thoughts on updating these variants to be '12-hour' and '24-hour'?

Reply:
Updated here. https://github.com/cerner/terra-core/pull/1027/commits/ebf13a984c6eae72602c245b3b548232da16ff22